paymentsheet/src/main/java/com/stripe/android/paymentsheet/elements/country/CountryConfig.kt

```diff
@@ -5,12 +5,21 @@ import com.stripe.android.paymentsheet.R
 import com.stripe.android.paymentsheet.elements.common.DropdownConfig
 import java.util.Locale
 
-internal class CountryConfig(val locale: Locale = Locale.getDefault()) : DropdownConfig {
+internal class CountryConfig(
+    val onlyShowCountryCodes: Set<String> = emptySet(),
+    val locale: Locale = Locale.getDefault()
+) : DropdownConfig {
     override val debugLabel = "country"
 
     @StringRes
     override val label = R.string.address_label_country
 
-    override fun getDisplayItems(): List<String> =
-        CountryUtils.getOrderedCountries(locale).map { it.name }
+    override fun getDisplayItems(): List<String> = if (onlyShowCountryCodes.isNotEmpty()) {
+        CountryUtils.getOrderedCountries(locale)
+            .filter { onlyShowCountryCodes.contains(it.code.value) }
+            .map { it.name }
+    } else {
+        CountryUtils.getOrderedCountries(locale)
+            .map { it.name }
+    }
```

Comment:
nit: this could be condensed into 
```
    override fun getDisplayItems(): List<String> =
        CountryUtils.getOrderedCountries(locale)
            .filter {
                onlyShowCountryCodes.isEmpty() || onlyShowCountryCodes.isNotEmpty() && onlyShowCountryCodes.contains(it.code.value)
            }
            .map { it.name }
```

Also we're depending on the emptiness of `onlyShowCountryCodes` to decide if we want to filter or not, probably worthwhile adding a comment to this parameter to clarify this